opts/options.go

```diff
@@ -1,271 +1,80 @@
+// Deprecated: Options are now defined in the root package.
+
 package dhtopts
 
 import (
-	"fmt"
 	"time"
 
 	ds "github.com/ipfs/go-datastore"
-	dssync "github.com/ipfs/go-datastore/sync"
 	"github.com/libp2p/go-libp2p-core/protocol"
+	"github.com/libp2p/go-libp2p-kad-dht"
 	"github.com/libp2p/go-libp2p-record"
 )
 
 // Deprecated: The old format did not support more than one message per stream, and is not supported
 // or relevant with stream pooling. ProtocolDHT should be used instead.
-const ProtocolDHTOld protocol.ID = "/ipfs/dht"
+const ProtocolDHTOld = dht.ProtocolDHTOld
```

Comment:
Do we need this at all? I was thinking of removing it in #479, could kill it there since it might be more appropriate.

Reply:
Note: we're going to have to refactor this anyways when we fix protocol string handling.